### PR TITLE
Rss testing

### PR DESCRIPTION
**Pull request by ERetana2**
Test RSS feed with new objects.  Current output:

xml feed string

### CODE REVIEW

File: launchpod/src/main/java/com/google/launchpod/data/Item.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.google.launchpod.data;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "item")
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/Item.java]: try getting rid of this

File: launchpod/src/main/java/com/google/launchpod/data/Channel.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.google.launchpod.data;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "channel")
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/Channel.java:11]: try getting rid of this

File: launchpod/src/main/java/com/google/launchpod/servlets/FormHandlerServlet.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    Key urlID = KeyFactory.stringToKey(id);
+    // Search key in datastore
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    // create entity that contains id from datastore
+    Entity desiredFeedEntity;
+    try {
+      desiredFeedEntity = datastore.get(urlID);
+      // generate xml string
+      UserFeed desiredUserFeed = UserFeed.fromEntity(desiredFeedEntity);
+      RSS rssFeed = new RSS(desiredUserFeed.getTitle(), desiredUserFeed.getLink(), desiredUserFeed.getPubDate());
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/servlets/FormHandlerServlet.java:98]: can you double check that desiredUserFeed.getTitle() and the other fields aren't empty?

Comment (ERetana2) [reply launchpod/src/main/java/com/google/launchpod/servlets/FormHandlerServlet.java:98]: Just checked, they do contain the input from the form

File: launchpod/src/main/java/com/google/launchpod/data/Channel.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+public final class Channel {
+
+  @JacksonXmlProperty
+  private final String title = "Launchpod";
+
+  @JacksonXmlProperty
+  private final String link = "https://launchpod-step18-2020.appspot.com";
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/Channel.java]: we might need to make these all variables non-final as well

File: launchpod/src/main/java/com/google/launchpod/data/Channel.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package com.google.launchpod.data;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "channel")
+public final class Channel {
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/Channel.java]: no need to make this final

File: launchpod/src/main/java/com/google/launchpod/data/Item.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+public class Item {
+  
+  @JacksonXmlProperty
+  private final String title;
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/Item.java]: don't make final

File: launchpod/src/main/java/com/google/launchpod/data/RSS.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+package com.google.launchpod.data;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "rss")
+public class RSS {
+    
+  @JacksonXmlProperty
+  private final Channel channel;
```
Comment (qihuang2) [inline launchpod/src/main/java/com/google/launchpod/data/RSS.java]: don't make final

File: launchpod/src/main/java/com/google/launchpod/data/Item.java (hunk 1 above)
Comment (ERetana2) [reply launchpod/src/main/java/com/google/launchpod/data/Item.java]: getting an error   
 com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class com.google.sps.data.Channel and no properties discovered to create BeanSerializer

File: launchpod/src/main/java/com/google/launchpod/data/Channel.java (hunk 2 above)
Comment (ERetana2) [reply launchpod/src/main/java/com/google/launchpod/data/Channel.java:11]: same as above

**Pull request merged**